students/mariusz_michalczyk/lesson_04_unit_testing/TestDistance.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import The_length_of_the_segment
+
+
+class TestDistanceFunction(unittest.TestCase):
+    def test_type_error_when_string_entered(self):
+        with self.assertRaises(TypeError):
+            The_length_of_the_segment.distance("shit")
+
+    def test_type_error_when_none_entered(self):
+        with self.assertRaises(TypeError):
+            The_length_of_the_segment.distance(None)
+
+    def test_zero_length(self):
```

Comment:
This is not so much, it doesn't say what and when it fails, you will have to check the code if the test doesn't pass.